### PR TITLE
Add releases page to demo

### PR DESCRIPTION
**Pull request by rvsia**
Adds `Releases` page with data obtained by the GitHub's API

image

### CODE REVIEW

**Comment by Hyperkid123** [comment]
O would probably move it to the bottom of the nav or use the current version number as a link in the top right toolbar.

Also, I don't like how the links look :smile: Maybe use the same styling as in the rest of the docs.

**Comment by rvsia** [comment]
Updated: 
Screenshot from 2020-01-17 12-59-50

**Comment by Hyperkid123** [comment]
@rvsia I am looking at the dates and something does not seem to be right. Date 2020-00-15 just does not seem to be right :D

**Comment by rvsia** [comment]
@Hyperkid123 Yeah, it's not right, however: https://github.com/data-driven-forms/react-forms/releases ..   :skull_and_crossbones:

**Comment by Hyperkid123** [comment]
> @Hyperkid123 Yeah, it's not right, however: https://github.com/data-driven-forms/react-forms/releases ..

I would gues that you should create a date object from the string.

Months and days start with 0 in JS.

**Comment by rvsia** [comment]
@Hyperkid123 The problem is that GitHub provides these data in a string

example: 

```
"body": "\n# [1.26.0](https://github.com/undefined/compare/v1.25.2...v1.26.0) (2020-00-15)\n\n## Minor changes\n\n### [@data-driven-forms/pf3-component-mapper~1.26.0](https://www.npmjs.com/package/@data-driven-forms/pf3-component-mapper/v/1.26.0) \n * feat(renderer): remove parsers from renderer\n\n### [@data-driven-forms/pf4-component-mapper~1.26.0](https://www.npmjs.com/package/@data-driven-forms/pf4-component-mapper/v/1.26.0) \n * feat(renderer): remove parsers from renderer\n\n### [@data-driven-forms/react-form-renderer~1.26.0](https://www.npmjs.com/package/@data-driven-forms/react-form-renderer/v/1.26.0) \n * feat(renderer): remove parsers from renderer\n\n### [with-firebase-hosting~1.26.0](https://www.npmjs.com/package/with-firebase-hosting/v/1.26.0) \n * feat(renderer): remove parsers from renderer\n\n## Bug fixes\n\n### [@data-driven-forms/pf4-component-mapper~1.26.0](https://www.npmjs.com/package/@data-driven-forms/pf4-component-mapper/v/1.26.0) \n * Update snaps\n\n### [@data-driven-forms/react-form-renderer~1.26.0](https://www.npmjs.com/package/@data-driven-forms/react-form-renderer/v/1.26.0) \n * Remove resetAble\n * Update readmes\n * Removed unused formType prop\n\n### [with-firebase-hosting~1.26.0](https://www.npmjs.com/package/with-firebase-hosting/v/1.26.0) \n * Add transform plugin babel to demo\n\n  "
```

**Comment by Hyperkid123** [comment]
Ugh. Maybe we can change the bot release message?

**Comment by rvsia** [comment]
> Ugh. Maybe we can change the bot release message?

Yep.

The problem is here: https://github.com/karelhala/commit-analyzer-wildcard/blob/057570b5456b39880cd90f10188a3dc94292484f/packages/release-notes/lib.js#L53 getMonth returns 0...

I will open a PR there.

**Comment by Hyperkid123** [comment]
@rvsia days should return 0 aswell

**Comment by rvsia** [comment]
@Hyperkid123 

```javascript
const date = new Date('1-1-2020');

date.getDate() // <- 1
date.getMonth() // <- 0
```

Javascript :man_facepalming:

**Comment by Hyperkid123** [comment]
lol

**Comment by rvsia** [comment]
@Hyperkid123 rebased

File: packages/react-renderer-demo/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
         "firebase-admin": "^8.5.0",
         "firebase-functions": "^2.1.0",
         "grpc": "^1.22.2",
+        "markdown-to-jsx": "^6.10.3",
```
Comment (Hyperkid123) [inline packages/react-renderer-demo/package.json]: This should not be needed actually. We already have MDX.

**Pull request merged**